### PR TITLE
fix: random collections populating fairs pages

### PR DESCRIPTION
**Pull request by egdbear**
This PR fixes the issue of random collections populating as curated highlights on the fair page when the admin field is blank.

[Ticket](https://artsyproduct.atlassian.net/browse/DIA-786)

### CODE REVIEW

File: src/schema/v2/fair.ts [hunk 1]
```diff
@@ -227,7 +227,12 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
             args,
             { marketingCollectionsLoader }
           ) => {
-            args.slugs = kaws_collection_slugs
+            args.slugs = kaws_collection_slugs || []
```
Comment (araujobarret) [inline src/schema/v2/fair.ts:230]: I wouldn't mutate the args.slugs, I'd rather use a new variable to avoid any side effects

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: good one

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: wow my connection is really bad

Comment (egdbear) [reply src/schema/v2/fair.ts:230]: wow my connection is really bad

**Review by araujobarret — APPROVED** [review]

**Review by araujobarret — APPROVED** [review]

**Review by araujobarret — APPROVED** [review]

**Pull request merged**